src/components/operations/operation-details/ko/runtime/operation-console.ts

```diff
@@ -491,8 +493,11 @@ export class OperationConsole {
                     this.responseBody(Utils.formatXml(this.responseBody()));
                 }
             }
+
+            this.logSentRequest(this.api().id, consoleOperation.id, method, response.statusCode.toString());
```

Comment:
One more case where `id` property will be gone: `this.api().id`. Please use `name` instead (not to be confused with `displayName`). Same goes to any other resource.

Reply:
Update to name, will keep in mind that.